src/cts/src/TechChar.cpp

```diff
@@ -451,18 +444,10 @@ void TechChar::initCharacterization()
 {
   odb::dbChip* chip = db_->getChip();
   odb::dbBlock* block = chip->getBlock();
-  float dbUnitsPerMicron = block->getDbUnitsPerMicron();
-
+  int dbUnitsPerMicron = block->getDbUnitsPerMicron();
```

Comment:
Need to be careful because `getClockLayerResCap()` now takes in `int`, not `float`

Reply:
static cast to float will do the trick, right?